src/components/student/ItemCard.tsx

```diff
@@ -0,0 +1,103 @@
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Card from '@mui/material/Card'
+import CardActions from '@mui/material/CardActions'
+import CardContent from '@mui/material/CardContent'
+import Typography from '@mui/material/Typography'
+import { Classroom, Item, Player } from '../../types'
+import { capitalize } from 'lodash'
+import { purchaseItem, updateAvatar } from '../../utils/mutations'
+import { useState } from 'react'
+
+import { styled } from '@mui/system'
+
+// Props
+interface Props {
+	item: Item
+	player: Player
+	classroom: Classroom
+	itemPrice: string
+	type: 'shop' | 'inventory'
+}
+
+// The display for a card
+export function ItemCard(props: Props) {
+	const [text, setText] = useState('')
+
+	console.log('item')
+	console.log(props.item)
+
+	// Styled box
+	const ItemBox = styled(Box)({
+		top: -20,
+		left: '18%',
+		position: 'relative',
+		width: '150px',
+		height: '120px',
+	})
+
+	// Styled typography
+	const ItemTypography = styled(Typography)({
+		marginTop: '15px',
+		color: '#5c5c5c',
+	})
+
+	// Purchasing
+	const handlePurchase = async () => {
+		const res = (await purchaseItem(props.classroom.id, props.player.id, props.item)) || null
```

Comment:
This line can be simplified by the nullish coalescing operator (??, look it up). This destroyed me in an interview so it's probably helpful for you to know it LOL